modules/social_features/social_branding/tests/src/Kernel/GraphQL/BrandingGraphQLTest.php

```diff
@@ -0,0 +1,384 @@
+<?php
+
+namespace Drupal\Tests\social_branding\Kernel\GraphQL;
+
+use Drupal\Tests\social_graphql\Kernel\SocialGraphQLTestBase;
+use Drupal\social_branding\Wrappers\Color;
+
+/**
+ * Base class for branding related GraphQL tests.
+ */
+class BrandingGraphQLTest extends SocialGraphQLTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    "social_branding",
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $configSchemaCheckerExclusions = [
+    'bootstrap.settings',
+    // Delete it from exclusions when the schema is added.
+    // @see https://www.drupal.org/project/socialbase/issues/3221046
+    // @see https://www.drupal.org/project/socialblue/issues/3221047
+    'socialbase.settings',
+    'socialblue.settings',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() : void {
+    parent::setUp();
+
+    // Use the socialblue theme.
+    $this->container
+      ->get('theme_installer')
+      ->install([
+        'socialblue',
+      ]);
+    $this->container
+      ->get('config.factory')
+      ->getEditable('system.theme')
+      ->set('default', 'socialblue')
+      ->save();
+  }
+
+  /**
+   * Ensure the platform branding fields are properly added to the endpoint.
+   */
+  public function testPlatformBrandingFieldsPresence() : void {
```

Comment:
A possible future improvement would be to fill the config in the test, without enabling a theme. That should speed up the test and decouples it better from the theming layer. But this can be done when the branding is refactored :)